consul-connect/provision.code.sh

```diff
@@ -0,0 +1 @@
+while [ ! -f /tmp/provision.sh ]; do sleep 1; done; /tmp/provision.sh
```

Comment:
I'd like to avoid having to do this if we can build a better disk image to start.